media-proxy/src/libfabric_ep.c

```diff
@@ -264,6 +268,7 @@ int ep_init(ep_ctx_t **ep_ctx, ep_cfg_t *cfg)
         if (ret)
             return ret;
     }
+    (*ep_ctx)->mr_access = rdma_info_to_mr_access(fi);
```

Comment:
(*ep_ctx)->mr_access is used only in one place, maybe it is better to make this variable local(instead of struct member) ?

Reply:
I removed mr_access from ep_ctx_t, I wasn't sure if I could calculate it from the data associated with libfabric_ctx, but it seems to work.